functions/Remove-DbaPlanCache.ps1

```diff
@@ -0,0 +1,89 @@
+﻿function Remove-DbaPlanCache {
+    <#
+        .SYNOPSIS
+            Removes adhoc and prepared plan caches is single use plans are over defined 
+            threshold
+    
+        .DESCRIPTION
+            "Checks ahoc and prepared plan cache for each database, if over 100 MBs 
+            removes from the cache."
+    
+            This command automates that process.
+    
+            References: https://www.sqlskills.com/blogs/kimberly/plan-cache-adhoc-workloads-and-clearing-the-single-use-plan-cache-bloat/
+    
+            Note: This command removes the plans from all SQL instances on the destionation 
+            server but the process column is specific to -SqlInstance passed.
+    
+        .PARAMETER SqlInstance
+            Allows you to specify a comma separated list of servers to remove the plans.
+    
+        .PARAMETER SqlCredential
+            Allows you to login to the SQL instance using alternative credentials.
+    
+        .PARAMETER Threshold
+            Memoery used threshold.
```

Comment:
Spelling correction: `Memory`

Do we also want to add here that the unit of measure is MB?